pkg/hubble/server/server.go

```diff
@@ -32,7 +32,7 @@ import (
 
 var (
 	errNoListener             = errors.New("no listener configured")
-	errNoTransportCredentials = errors.New("no transport credentials set (use serveroption.WithInsecure() or set credentials)")
+	errNoTransportCredentials = errors.New("no transport credentials set (use WithInsecure() or set credentials)")
```

Comment:
This is a message that is going to be seen by users. right? How can they know which flag sets the `WithInsecure()`?

Reply:
I'm not sure if this error message is logged and thus users can see it (I have to check). I think I can just truncate the message then.